### PR TITLE
Prevent orphans when wrapping the masthead title

### PR DESCRIPTION
**Pull request by cbeer**
Before:
Screen Shot 2022-12-15 at 13 25 01

After:
Screen Shot 2022-12-15 at 13 24 23

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 en:
   arclight:
-    masthead_heading: Taube Archive of the International Military Tribunal (IMT) at Nuremberg, 1945-46
+    masthead_heading: Taube Archive of the International Military Tribunal (IMT) at Nuremberg,&nbsp;1945&#x2011;46
```
Comment (jcoyne) [inline config/locales/en.yml:3]: Would it make sense to move this to arclight in `masthead_heading_html` so we wouldn't need to `html_safe`?

Comment (cbeer) [reply config/locales/en.yml:3]: I assume that key comes from upstream arclight?

**Comment by marlo-longley** [comment]
Ha! I just noticed that was my demo screen for the demo. Thank you Chris.

File: config/locales/en.yml (hunk 1 above)
Comment (jcoyne) [reply config/locales/en.yml:3]: Yes, I mean change https://github.com/projectblacklight/arclight/blob/47a388424b7357be35e600258b05415e0b22ad7a/app/components/arclight/masthead_component.rb#L7 and https://github.com/projectblacklight/arclight/blob/20aa68e749a06e504ede65abd6cf05cf70b0301f/config/locales/arclight.en.yml#L14 ?

**Review by marlo-longley — APPROVED** [review]

**Comment by jcoyne** [comment]
Can we do this instead? https://github.com/projectblacklight/arclight/pull/1374

**Pull request closed without merging**